pom.xml

```diff
@@ -28,14 +28,14 @@
   <groupId>com.google.googlejavaformat</groupId>
   <artifactId>google-java-format-parent</artifactId>
   <packaging>pom</packaging>
-  <version>1.8-SNAPSHOT</version>
+  <version>${revision}</version>
```

Comment:
This will break deployments to Maven repositories unless you throw one more plugin to it: https://maven.apache.org/maven-ci-friendly.html#install-deploy